CriteoPublisherSdk/Tests/UnitTests/Network/CR_NetworkManagerTests.m

```diff
@@ -138,7 +135,7 @@ - (void)skipped_testNetworkManagerPostCall {
                                nil],
           @"slots", nil];
 
-  NSURL *url = [NSURL URLWithString:@"http://directbidder-test-app.par.preprod.crto.in/"
+  NSURL *url = [NSURL URLWithString:@"https://localhost:9099/directbidder-test-app/"
                                     @"inapp/v1?profileId=235"];
```

Comment:
/inapp/v1 ?

I'm afraid that this test is completely wrong :D
AFAIK the /inapp/v1 endpoint of CDB is for replying to the Publisher SDK v1 that was made out of Criteo 2~3 years ago.  This publisher SDK should use the /inapp/v2 endpoint.

What do you think about trying to migrate this test to use the v2 endpoint ? It may look FAR, but I don't think it is a good idea neither to have such kind of test that brings nothing to this SDK.
The payload might have change a little bit.

Reply:
Done. But as commented in commit, kind of useless as this test only covers the NetworkManager post code w/o the business logic, which is tested elsewhere. This is an integration test w/o mocking the networking call.